src/Symfony/Component/Messenger/Transport/AmqpExt/AmqpFactory.php

```diff
@@ -11,25 +11,19 @@
 
 namespace Symfony\Component\Messenger\Transport\AmqpExt;
 
-class AmqpFactory
-{
-    public function createConnection(array $credentials): \AMQPConnection
-    {
-        return new \AMQPConnection($credentials);
-    }
+use Symfony\Component\Messenger\Bridge\AmqpExt\Transport\AmqpFactory as BridgeAmqpFactory;
 
-    public function createChannel(\AMQPConnection $connection): \AMQPChannel
-    {
-        return new \AMQPChannel($connection);
-    }
+if (! class_exists(BridgeAmqpFactory::class, false)) {
+    @trigger_error(sprintf('The "%s" class is @deprecated since Symfony 5.1, use "%s" instead. The AmqpExt transport has been moved to package "symfony/messenger-amqp" and will not be required by default in 6.0. Run "composer require symfony/messenger-amqp".', AmqpFactory::class, BridgeAmqpFactory::class), E_USER_DEPRECATED);
```

Comment:
I don't think we use a `@` before deprecated

Maybe it's just me, but the sentense `will not be required by default` may be understood like "the pakage will not be necessary" which is the oposite. 
What's about `will not be provided in 6.0` or `included`

Reply:
Oops, thank you. 

I think "included" sounds better. I used "required" because it is composer terminology.